src/any_llm/providers/gemini/utils.py

```
@@ -276,6 +276,18 @@ def _extract_usage_dict(response: types.GenerateContentResponse) -> dict[str, An
     return usage
 
 
+def _thought_signature_extra_content(part: Any) -> dict[str, Any] | None:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `part`

(ANN401)